<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessItem</class>
 <widget class="QWidget" name="ProcessItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>434</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>40</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QTableView" name="PCBInfo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_unsuspend">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>解除阻塞</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_suspend">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>阻塞</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
